docs/source/pages/developers_guide/index.rst

```diff
@@ -149,9 +149,8 @@ provide the check details in the initializer method, and implement the logic of
 
 A ``check_id`` should meet the following requirements:
 
-    - The general format: ``mcn_<name>_<digits>``
-    - In ``name``, only lowercase alphabetical letters are allowed. If ``name`` contains multiple \
-        words, they must be separated by underscores.
+    - The general format: ``mcn_<name>_<digits>``.
+    - Lowercase alphabetical letters are allowed in ``name`` only. If ``name`` contains multiple words, they must be separated by underscores.
```

Comment:
Sorry for nitpicking but this sounds more like "the only part of the check id where lowercase alphabetical letters are allowed is `name`", which is different from "Only lowercase alphabetical letters are allowed in ``name``."

Reply:
When `only` is used at the end of a sentence, it's a modifier for the whole sentence. Here I don't want to focus on `name` or `lowercase`.

Reply:
I agree with Nathan's interpretation of the literal meaning of the sentence. However, "Only lowercase alphabetical letters are allowed in `name`" would also be incorrect because it is immediately contradicted by the following statement that implies that underscore is also allowed.

Presuming that the intention is to convey that name has the form `[a-z]+(_[a-z]+)+`, a precise description would be "`name` must consist solely of words (sequences of lowercase alphabetical letters) separated by underscores" (you could even include the regex if you want to be absolutely precise).

(To be extra nitpicky, "in name only" is a such a common phrase with a specific meaning that I'd avoid invoking it by coincidence, I know that `name` is a specially-quoted identifier that shouldn't be read literally, but it still throws me a little)

Reply:
See commit [08eb1ce](https://github.com/oracle/macaron/pull/620/commits/08eb1ce77c142753c5fae77b68d5b2b765c50e3d).